components/nimbus/src/tests/stateful/test_evaluator.rs

```diff
@@ -241,7 +241,7 @@ fn test_event_transform_last_seen_parameters() {
             &th,
         ),
         Some(EnrollmentStatus::Error {
-            reason: "EvaluationError: Custom error: JSON Error: invalid type: floating point `1`, expected a string"
+            reason: "EvaluationError: Custom error: JSON Error: invalid type: floating point `1.0`, expected a string"
```

Comment:
@jeddai Looks like the fallout was just the error strings that `serde` sent out.  Any concerns with these test changes?

Reply:
Nope!